src/components/custom-widget/customWidgetContract.ts

```diff
@@ -0,0 +1,21 @@
+import { LocalStyles } from "@paperbits/common/styles";
+import { Contract } from "@paperbits/common";
+
+export interface CustomWidgetContract extends Contract {
+    /**
+     * This name is used to identify which custom-widget-binder to load into iframe.
+     */
+    name: string;
+
+    /**
+     * @deprecated. This name will be shown in editors and custom-widget-binder selector. (this will be removed from here and used only in WidgetHandlers)
+     */
+    widgetDisplayName: string;
```

Comment:
@azaslonov you've marked the widgetDisplayName as deprecated and only to be used in WidetHandlers. As I'm using it even elsewhere, I would like to keep this property here if possible.